openrtb_ext/imp_appnexus_Keywords_test.go

```diff
@@ -7,29 +7,27 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestKeywordsUtilUnmarshalJSON(t *testing.T) {
+func TestKeywordsUnmarshalJSON(t *testing.T) {
 	type keywords struct {
 		Keywords Keywords `json:"keywords"`
 	}
 
-	t.Run("dynamic-json", func(t *testing.T) {
-		jsonData := []byte(`{"keywords" : { "pets": ["dog"] }}`)
-		var keywords keywords
-		assert.NoError(t, json.Unmarshal(jsonData, &keywords))
-		assert.Equal(t, "pets=dog", keywords.Keywords.String())
-	})
+	type validTest struct {
+		input    []byte
```

Comment:
Thanks for converting these to table driven tests. 

Please consider adding a description field to this struct and you can then pass the description to the assert calls as the last argument like:
```
assert.NoError(t, json.Unmarshal(test.input, &keywords), test.desc)
```
This prints out the test description in case the test case fails and helps to easily identify which test case is having issues.